docs/team/alexwu.adoc

```diff
@@ -0,0 +1,71 @@
+= Alex Wu - Project Portfolio
```

Comment:
File name should be your github name in lowercase https://github.com/AY1920S1-CS2103T-W13-1/main/issues/176

Reply:
sorry abt that. i fixed it